docs/webpack-dev-server.config.js

```diff
@@ -51,14 +51,7 @@ const config = {
   module: {
     //eslint loader
     preLoaders: [
-      {
-        test: /\.(js|jsx)$/,
-        loader: 'eslint-loader',
-        include: [path.resolve(__dirname, '../src')],
-        exclude: [
-          path.resolve(__dirname, '../src/svg-icons'),
-        ],
-      },
+
```

Comment:
I decided to put it back, and I'll kill it in a separate PR, as we reference it in `contributing.md`, so a separate PR will be cleaner as I can edit that at the same time. When I do, should I remove the whole `preloaders` section, or leave it empty for future reference?